lib/cli.js

```diff
@@ -536,3 +532,16 @@ function handleError(err) {
 
 	process.exit(exitCode); // eslint-disable-line no-process-exit
 }
+
+/**
+ * @param {string[]} argv
+ * @returns {CLIOptions}
+ */
+function buildCLI(argv) {
+	meowOptions.argv = argv;
```

Comment:
Maybe instead of mutating `meowOptions` we should do:

```js
return meow({ ...meowOptions, argv });
```

Reply:
Make sense. I fixed via 668bbf6a971e7949e9603fa3f51eb40680546370. 👌